Docs/index.md

```diff
@@ -0,0 +1,46 @@
+# Welcome to the Santa Docs
+
+Santa is a binary whitelisting/blacklisting system for macOS. Here you will find the documentation for understanding how Santa works, how to deploy it and how to contribute.
+
+#### Introduction
+
+The following documents give an overview on how Santa accomplishes binary whitelisting/blacklisting at the enterprise scale.
+
+- [Binary Whitelisting](introduction/binary-whitelisting-overview.md): How Santa makes allow or deny decisions for any `exec()` taking place.
+- [Syncing](introduction/syncing-overview.md): How configuration and whitelist / blacklist rules are applied from a sync-server.
+
+#### Deployment
+
+* [Configuration](deployment/configuration.md): The local and sync-server configuration options.
+
+#### Development
+
+* [Building Santa](development/building.md): How to build and load test builds of Santa on a development machine.
```

Comment:
[Building Santa](development/building.md): How to build and load Santa for testing on a development machine.